src/Nordea/Components/AccordionMenu.elm

```diff
@@ -0,0 +1,63 @@
+module Nordea.Components.AccordionMenu exposing (OptionalConfig(..), Status(..), content, header, view)
+
+import Css exposing (alignItems, center, display, displayFlex, float, inlineBlock, justifyContent, lastChild, listStyle, none, nthLastChild, rem, right, spaceBetween, width)
+import Css.Global exposing (children, typeSelector, withAttribute)
+import Html.Styled as Html exposing (Attribute, Html, div)
+import Html.Styled.Attributes exposing (attribute, css)
+import Nordea.Resources.Icons as Icons
+
+
+type Status
+    = Open
+    | Closed
+
+
+type OptionalConfig
+    = Status Status
+
+
+view : List OptionalConfig -> List (Attribute msg) -> List (Html msg) -> Html msg
+view config attrs children_ =
```

Comment:
Since the functionality of if the accordion is open or not is so integral to how this component works, I think it would better be represented by a record where the value is mandatory rather than a list of optional arguments 😊 We could also consider representing it with a `Bool` since it will always be just open or closed. 
What do you think? 😊
```suggestion
type alias Config =
     { isOpen : Bool }

view : Config -> List (Attribute msg) -> List (Html msg) -> Html msg
view config attrs children_ =
```

Reply:
Yes, making it a mandatory field could be a better idea. I will do it.